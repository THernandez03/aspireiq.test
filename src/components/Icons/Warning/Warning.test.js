import React from "react";
import { render } from "@testing-library/react";

import { Warning } from "./Warning";

describe("<Warning/>", () => {
  it("should render a Warning", () => {
    const { container } = render(<Warning />);
    expect(container).toMatchSnapshot();
  });
});
