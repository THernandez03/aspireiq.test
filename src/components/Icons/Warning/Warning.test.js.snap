// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Warning/> should render a Warning 1`] = `
<div>
  <svg
    fill="currentColor"
    height="1em"
    stroke="currentColor"
    stroke-width="0"
    viewBox="0 0 24 24"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <path
        d="M10 6H14v12H10Z"
        fill="currentColor"
      />
      <path
        d="M12 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10zm-1-7v2h2v-2h-2zm0-8v6h2V7h-2z"
      />
    </g>
  </svg>
</div>
`;
